From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Wed, 25 Mar 2020 18:08:39 +0000
Subject: dt: Add HDMI audio dma values to bcm2711-rpi.dtsi

Adds the relevant DMA settings for HDMI audio to work.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2711-rpi.dtsi b/arch/arm/boot/dts/bcm2711-rpi.dtsi
index aa58ddd51ff7..f8a75550c9b4 100644
--- a/arch/arm/boot/dts/bcm2711-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2711-rpi.dtsi
@@ -73,6 +73,8 @@
 			clock-names = "hdmi";
 			resets = <&dvp 0>;
 			ddc = <&ddc0>;
+			dmas = <&dma 10>;
+			dma-names = "audio-rx";
 			status = "disabled";
 		};
 
@@ -108,6 +110,8 @@
 			clocks = <&firmware_clocks 13>;
 			clock-names = "hdmi";
 			resets = <&dvp 1>;
+			dmas = <&dma 17>;
+			dma-names = "audio-rx";
 			status = "disabled";
 		};
 
